fhir-bulkimportexport/src/main/java/com/ibm/watson/health/fhir/bulkexport/ChunkReader.java

```diff
@@ -168,14 +188,12 @@ public Object readItem() throws Exception {
             queryString += ("&_lastUpdated=ge" + fhirSearchFromDate);
```

Comment:
The following block is duplicate of the other `readItem`

Reply:
It's actually in the batchlet job, I didn’t de-duplicate this because I may need 2 functions - one for queryString and one for queryParameters, it doesn’t like python or go which can return multiple,  or I have to create a class to hold both and return the object from the class … seems a little too much for this simple purpose.